core/http/client/src/main/java/org/eclipse/rdf4j/http/client/RDF4JProtocolSession.java

```diff
@@ -621,8 +621,12 @@ public synchronized void beginTransaction(IsolationLevel isolationLevel)
 
 			List<NameValuePair> params = new ArrayList<>();
 			if (isolationLevel != null) {
-				params.add(new BasicNameValuePair(Protocol.ISOLATION_LEVEL_PARAM_NAME,
-						isolationLevel.getURI().stringValue()));
+				params.add(
+					new BasicNameValuePair(
+						Protocol.ISOLATION_LEVEL_PARAM_NAME,
+						isolationLevel.getURI().stringValue()
+					)
+				);
```

Comment:
example of how parentheses are now allowed to be indented/wrapped

Reply:
Also note that everything that wraps is now indented with a single indent, instead of two indents. Hence the big diff.